godot-core/src/builtin/plane.rs

```diff
@@ -105,6 +107,90 @@ impl Plane {
         }
     }
 
+    /// Returns the shortest distance from the `self` `Plane` to the provided `Vector3`. The distance 
+    /// will be positive if the `Vector3` is above the `Plane`, and will be negative if the `Vector3` 
+    /// is below the `Plane`.
+    ///
+    /// _Godot equivalent: `Plane.distance_to(Vector3 point)`_
```

Comment:
The "Godot equivalent" section is not necessary for all methods, but only those that are:
* either named differently (more than just omitting a `get_` prefix)
* or have a different signature (function parameter types and order).

In cases like this, you can omit it.